extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -371,34 +372,4 @@ public static function doRetroactiveAutoblock( Block $block, array &$blockIds )
 
 		return false; // autoblock handled
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app